packages/cannon-worker-api/src/worker/operations/init.ts

```diff
@@ -39,6 +40,9 @@ export const init = (
 ): void => {
   world.allowSleep = allowSleep
   world.gravity.set(...gravity)
+  if (frictionGravity) {
+    world.frictionGravity = new Vec3(...frictionGravity)
+  }
```

Comment:
I prefer consistently assigning values:

```suggestion
  world.frictionGravity = frictionGravity ? new Vec3(...frictionGravity) : frictionGravity
```